src/Form/FormLabel.js

```diff
@@ -4,7 +4,8 @@ import classNames from 'classnames';
 import withStyles from '../styles/withStyles';
 
 export const styles = theme => {
-  const focusColor = theme.palette.primary[theme.palette.type === 'light' ? 'A700' : 'A200'];
+  // FIXME: 'dark' & 'light' were accent colors before the theme change
```

Comment:
https://material.io/guidelines/components/text-fields.html#text-fields-states

![capture d ecran 2018-01-09 a 08 12 53](https://user-images.githubusercontent.com/3165635/34708889-2cca88a0-f515-11e7-908b-183d49b382fa.png)

Reply:
I think that we are good, you can remove the comment:
![capture d ecran 2018-01-09 a 08 18 14](https://user-images.githubusercontent.com/3165635/34708997-acc58eb0-f515-11e7-9553-5b8e386b8c6b.png)